src/test/java/com/gistpetition/api/petition/PetitionServiceTest.java

```diff
@@ -213,12 +209,31 @@ void retrieveRevisionsOfPetition() {
         PageRequest pageRequest = PageRequest.of(0, 10);
         Page<PetitionRevisionResponse> revisionResponses = petitionService.retrieveRevisionsOfPetition(petitionId, pageRequest);
         assertThat(revisionResponses.getContent()).hasSize(2);
-        assertThat(revisionResponses.getContent()).allMatch(content -> content.getWorkedBy() == petitionOwner.getId());
+        assertThat(revisionResponses.getContent()).allMatch(content -> content.getWorkedBy().equals(petitionOwner.getId()));
         List<PetitionRevisionResponse> content = revisionResponses.getContent();
         List<RevisionMetadata.RevisionType> revisionTypes = content.stream().map(PetitionRevisionResponse::getRevisionType).collect(Collectors.toList());
         assertThat(revisionTypes).containsExactly(RevisionMetadata.RevisionType.INSERT, RevisionMetadata.RevisionType.UPDATE);
     }
 
+    @Test
+    void retrieveTempPetitions() {
+        Long petitionId = petitionService.createPetition(DORM_PETITION_REQUEST, petitionOwner.getId());
+        User user1 = userRepository.save(new User("user1@gist.ac.kr", "pwd", UserRole.USER));
```

Comment:
```suggestion
for (int i=0; i<Petition.REQUIRED_AGREEMENT_NUM; i++) {
        users.add(userRepository.save(new User(String.format("user%s@gist.ac.kr", i), "pwd", UserRole.USER)));
}
```

이렇게, REQUIRED_AGREEMENT_NUM 에 의존적인 테스트로 만드는게 어떨까요? 저 숫자가 바뀌면 테스트를 다시 짜야할 것 같네요

Reply:
좋은 방법 같습니다!
agree도 같은 방법으로 해결해볼 수 있을 것 같아요!

Reply:
동의합니다!